Bottom: ab17e4d95123751782ce5365e4a9b290e0ce7050
Top:    434b2a407274722b9f73054065bdae0acdc6a101
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:28:00 +0100

hid/gtk: Remove unused function in_draw_state()


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 9ea3207..86abad2 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -610,21 +610,6 @@ ghid_port_key_press_cb (GtkWidget * drawing_area,
   return handled;
 }
 
-static gboolean
-in_draw_state (void)
-{
-  if ((Settings.Mode == LINE_MODE
-       && Crosshair.AttachedLine.State != STATE_FIRST)
-      || (Settings.Mode == ARC_MODE
-	  && Crosshair.AttachedBox.State != STATE_FIRST)
-      || (Settings.Mode == RECTANGLE_MODE
-	  && Crosshair.AttachedBox.State != STATE_FIRST)
-      || (Settings.Mode == POLYGON_MODE
-	  && Crosshair.AttachedLine.State != STATE_FIRST))
-    return TRUE;
-  return FALSE;
-}
-
 gboolean
 ghid_port_button_press_cb (GtkWidget * drawing_area,
 			   GdkEventButton * ev, GtkUIManager * ui)
